agent/php_nrini.c

```diff
@@ -255,6 +255,28 @@ static void check_ini_int_min_max(int* var, int minval, int maxval) {
   }
 }
 
+/*
+ * @brief error handling for strtol() functionality
+ *
+ * @param val_p   pointer to store the parsed value
```

Comment:
```suggestion
 * @param val_p   pointer to store the parsed value; remains undefined when
 * function returns NR_FAILURE
```

Reply:
resolved in 84867ece87b76e35e2b05d1458f8706cbcc3cc40